tests/ut/test_get_vector.cc

```diff
@@ -58,22 +59,22 @@ TEST_CASE("Test Binary Get Vector By Ids", "[Binary GetVectorByIds]") {
             make_tuple(knowhere::IndexEnum::INDEX_FAISS_BIN_IVFFLAT, bin_ivfflat_gen),
             make_tuple(knowhere::IndexEnum::INDEX_HNSW, bin_hnsw_gen),
         }));
-        auto idx = knowhere::IndexFactory::Instance().Create(name);
-        if (!idx.HasRawData(metric_type)) {
-            return;
-        }
+        auto idx = knowhere::IndexFactory::Instance().Create(name, version);
         auto cfg_json = gen().dump();
         CAPTURE(name, cfg_json);
         knowhere::Json json = knowhere::Json::parse(cfg_json);
         auto train_ds = GenBinDataSet(nb, dim);
         auto ids_ds = GenIdsDataSet(nb, nq);
         REQUIRE(idx.Type() == name);
         auto res = idx.Build(*train_ds, json);
+        if (!idx.HasRawData(metric_type)) {
```

Comment:
this change should be caused by rebase, please change it back

Reply:
No, it is by design. `HasRawData` could return different value for some indexes when `retrieve_friendly`.